lib/internal/assert/calltracker.js

```diff
@@ -88,12 +88,17 @@ class CallTracker {
 
   verify() {
     const errors = this.report();
-    if (errors.length > 0) {
-      throw new AssertionError({
-        message: 'Function(s) were not called the expected number of times',
-        details: errors,
-      });
+    if (!errors.length) {
+      return;
     }
+    let message = 'Function(s) were not called the expected number of times';
+    if (errors.length === 1) {
+      message = errors[0].message;
+    }
```

Comment:
```suggestion
    const message = errors.length === 1 ?
      errors[0].message,
      'Functions were not called the expected number of times';
```

Reply:
your ternary expression was `a ? b , c` so was not able to commit the suggestion directly, thanks for the useful comment anyway!